analytics/CMakeLists.txt

```diff
@@ -78,6 +78,11 @@ set(ios_SRCS
 set(stub_SRCS
     src/analytics_stub.cc)
 
+# Source files used by the stub implementation.
```

Comment:
Should we delete the stub_SRCS above? And/or, update the comment? Do we want to do stuff here to strip out src/windows for Mac/Linux, or leave it up to the file?